Anytype/Sources/ServiceLayer/Object/TypesService/TypesService.swift

```diff
@@ -95,4 +110,11 @@ public final class TypesService: TypesServiceProtocol {
         
         return try await searchMiddleService.search(filters: filters, sorts: [sort], fullText: text)
     }
+    
+    func searchPinnedTypes(text: String, spaceId: String) async throws -> [ObjectType] {
+        let page = try typeProvider.objectType(uniqueKey: .page, spaceId: spaceId)
```

Comment:
This is temporary code?

Reply:
Yep. It is just hardcoded for this pr. And I will make real one later on